app/components/interview_bookings_component.rb

```diff
@@ -0,0 +1,48 @@
+# frozen_string_literal: true
+
+class InterviewBookingsComponent < ViewComponent::Base
+  def initialize(application_choice)
+    @application_choice = application_choice
+  end
+
+  def interviews
+    @application_choice.interviews.includes(:provider).order(date_and_time: :asc)
+  end
+
+  def formatted_time(interview)
+    interview.date_and_time.to_s(:govuk_date_and_time).squish
```

Comment:
Is there any value in using `squish` here? The formatted date and time should never be a multiline string.

Reply:
This was added to handle the fact that `:govuk_date_and_time` leaves extra spaces in the string if dealing with single-digit day or hour. It's not especially clear what's going on though - have removed and just added an extra space to the spec. Will raise a separate PR to fix the date format.